sql/moz-fx-data-shared-prod/pocket_dbt_facts/dataset_metadata.yaml

```
@@ -0,0 +1,16 @@
+friendly_name: Pocket DBT Fact Models
+description: |-
+  This dataset contains the DBT fact models for Pocket. These can be used for any Pocket reporting.
+  https://github.com/Pocket/pocket-dbt
+
+
+dataset_base_acl: restricted
+user_facing: false
```

If this is supposed to be the user-facing dataset you should set:
```suggestion
user_facing: true
```
otherwise you need to rename this to `pocket_dbt_facts_derived`

---

Oh whoops

---

Updated!